deploy/raft-storage-controller.yaml

```diff
@@ -29,23 +29,94 @@ spec:
                   type: string
                 replicas:
                   type: integer
-                  min: 1
-                  max: 9
+                  default: 1
                 partitionsPerCluster:
                   type: integer
-                  min: 1
-                  max: 1024
                   default: 1
+                volumeClaimTemplates:
```

Comment:
Can we just convert this to YAML just so it's clear and so we're sure we can use it in the Helm chart as well?

Reply:
I will figure that out. It looks like it is working in helm chart as well :)